src/pages/gd/Bridge/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { memo, useCallback, useMemo } from 'react'
+import { useSwitchNetwork } from '@gooddollar/web3sdk-v2'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { useApplicationTheme } from 'state/application/hooks'
+import {
+    KimaTransactionWidget,
+    KimaProvider,
+    FontSizeOptions,
+    ColorModeOptions,
+    ModeOptions,
+    DAppOptions,
+} from '@kimafinance/kima-transaction-widget'
+import '@kimafinance/kima-transaction-widget/dist/index.css'
+
+const Bridge = memo(() => {
+    const { library } = useActiveWeb3React()
+    const [theme] = useApplicationTheme()
+    const { switchNetwork } = useSwitchNetwork()
+
+    const successHandler = useCallback(() => {
+        console.log('Kima bridge success:')
+    }, [])
+
+    const errorHandler = useCallback((e) => {
+        console.log('Kima bridge error:', e?.message, e)
+    }, [])
```

Comment:
success / error handlers could be moved outside, as do not depend from any state vars

Reply:
@johnsmith-gooddollar What do you mean `outside`? Bridge component would be referenced by `Routes.tsx`, you mean move success /error handlers to that file?

Reply:
@jeancarrie-kima these callbacks dont have any dependencies or use any params from the component so their definition can be moved outside the component.

1. do we need to show a success message/pop for the user? or the widget does that already?
2. @johnsmith-gooddollar please add analytics to success/error

Didn't we already had analytics integrated here?